articles/client-platforms/ember2js/01-login.md

```diff
@@ -17,15 +17,16 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  githubUrl: 'https://github.com/auth0/auth0-ember-simple-auth/tree/master/examples/ember2-auth0-sample',
-  pkgOrg: 'auth0',
-  pkgRepo: 'auth0-ember-simple-auth',
+  pkgOrg: 'auth0-samples' 
```

Comment:
@mehreencs87 Missing comma at the end of the line, throws syntax error upon compilation.